wls-gui-admintool/src/composables/wahltag/wahltagService.ts

```diff
@@ -28,11 +37,13 @@ export default function useWahltagService() {
     try {
       const wahltagDtos = await adminWahltageAPI
         .getWahltage()
-        .then((response) => response.data);
+        .then((response) => returnUndefinedOnStatus204OrElseDate(response));
 
-      const wahltageGroupByDatum = groupWahltagDtosByWahltag(wahltagDtos);
-      result.push(...mapGroupedWahltagDtosToWahltage(wahltageGroupByDatum));
-      result.forEach((wahltag) => wahltag.events.sort(compareByNummerAsc));
+      if (wahltagDtos) {
+        const wahltageGroupByDatum = groupWahltagDtosByWahltag(wahltagDtos);
+        result.push(...mapGroupedWahltagDtosToWahltage(wahltageGroupByDatum));
+        result.forEach((wahltag) => wahltag.events.sort(compareByNummerAsc));
+      }
```

Comment:
ich hatte eher an sowas gedacht, dass der nutzer nicht denkt dass die anwendung nicht funktioniert wenn er keine wahltage bekommt
```suggestion
      } else {
        addNotification("Keine Wahltage gefunden.", "Warning");
      }
```

Reply:
aber das sieht er doch im DropDown? Ich könnte den [Text](https://vuetifyjs.com/en/api/v-autocomplete/#props-no-data-text) anpassen wenn es zu generisch ist.

Reply:
text anpassen klingt gut für mich 👍🏽

Reply:
angepasst in der Komponente und eine Story dazu ergänzt: 929eaa49176d07f66b27f76b7a751593e52d630a

neu:  
![grafik](https://github.com/user-attachments/assets/34378439-3605-4309-8cd9-1fed881f2598)


alt:  
![grafik](https://github.com/user-attachments/assets/189a44aa-c331-4a98-8e46-1e5bd00d3fd1)

Ich habe mich bewusst gegen den Punkt am Ende entschieden, da in der alten Meldung auch kein Punkt am Ende war und ich davon ausgehe dass es bei den anderen Texten von vuetify so sein wird.